### PR TITLE
Gradle upgrade

### PR DESCRIPTION
**Pull request by kschults**
Addresses https://github.com/mgk/s3-plugin/issues/18

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -43,34 +43,31 @@ task versionFile {
...
+      description = 'S3 upload and download plugin'
       displayName = 'Gradle S3 plugin'
-      tags = ['aws', 's3']
+      tags.set(['aws', 's3'])
     }
   }
 }
 
-task wrapper(type: Wrapper) {
-    gradleVersion = project.gradleVersion
```
Comment (kschults) [inline build.gradle:70]: This doesn't seem to work anymore. The version comes from the wrapper file itself. It's self-updating using `./gradlew wrapper --gradle-version <version>`

File: build.gradle [hunk 2]
```diff
@@ -43,34 +43,31 @@ task versionFile {
...
-}
-
-task checkExample(type: GradleBuild, dependsOn: uploadArchives) {
+task checkExample(type: GradleBuild) {
```
Comment (kschults) [inline build.gradle:70]: @mgk I haven't worked with publishing much, so I wasn't sure how to address this. The `uploadArchives` task has been removed. I think as it is, the `check` task will fail, but I'm not sure exactly how to get it running.

File: src/main/resources/META-INF/gradle-plugins/com.github.mgk.gradle.s3.properties [hunk 3]
```diff
@@ -1 +0,0 @@
-implementation-class=com.github.mgk.gradle.S3Plugin
```
Comment (kschults) [inline src/main/resources/META-INF/gradle-plugins/com.github.mgk.gradle.s3.properties:1]: This is provided in the plugin `create` block now.

**Comment by kschults** [comment]
@mgk I think I've figured out the testing, and I've updated the `README` to reflect what worked for me. I've gotten to the point where the tests are failing because S3 is giving a 403 Forbidden response. It looks like the tests pull the S3 bucket from an environment variable `$BUCKET`, but I don't have a bucket handy to play with.

**Comment by kschults** [comment]
@mgk I've gotten a bucket to test with, and the tests now pass. Would love if we could get this merged and released!

**Comment by mgk** [comment]
@kschults Thanks much.

**Pull request merged**